extensions/azurecore/src/azureResource/utils.ts

```diff
@@ -333,8 +333,9 @@ export async function getSelectedSubscriptions(appContext: AppContext, account?:
  * @param requestBody Optional request body to be used in PUT and POST requests.
  * @param ignoreErrors When this flag is set the method will not throw any runtime or service errors and will return the errors in errors array.
  * @param host Use this to override the host. The default host is https://management.azure.com
+ * @param requestHeaders Provide additional request headers
```

Comment:
Update to `customHeaders` or make customHeaders be requestHeaders instead (regardless, make sure all the params are the same for the other places this is used too)